sdk/core/azure-core-test/src/main/java/com/azure/core/test/utils/TestProxyUtils.java

```diff
@@ -110,6 +113,9 @@ public static void changeHeaders(HttpRequest request, URL proxyUrl, String xReco
                 headers.set(X_RECORDING_UPSTREAM_BASE_URI, originalUrl.toString());
                 headers.set(X_RECORDING_MODE, mode);
                 headers.set(X_RECORDING_ID, xRecordingId);
+                if (mode.equals(RECORD_MODE) && skipRecordingRequestBody) {
+                    headers.set(X_RECORDING_SKIP, "request-body");
```

Comment:
Is there an option to record the checksum of the request body instead of the whole request body? We still want to validate the integrity of the request body but just not store it in the recording.

Reply:
I don't think we currently have that ability with Test Proxy.
@scbedd - do you know if we support this in Test Proxy yet.

Reply:
https://github.com/Azure/azure-sdk-tools/issues/6074 👍😄